src/python/pants/option/options_integration_test.py

```diff
@@ -1,315 +1,96 @@
-        self.assertNotIn("options.name = None", pants_run.stdout)
-        self.assertIn("pytest.timeouts = ", pants_run.stdout)
-
-    def test_valid_json(self) -> None:
-        pants_run = self.run_pants(["options", "--output-format=json"])
-        pants_run.assert_success()
-        try:
-            output_map = json.loads(pants_run.stdout)
-            self.assertIn("time", output_map)
-            self.assertEqual(output_map["time"]["source"], "HARDCODED")
-            self.assertEqual(output_map["time"]["value"], False)
-        except ValueError:
-            self.fail("Invalid JSON output")
-
-    def test_valid_json_with_history(self) -> None:
-        pants_run = self.run_pants(["options", "--output-format=json", "--show-history"])
-        pants_run.assert_success()
-        try:
-            output_map = json.loads(pants_run.stdout)
-            self.assertIn("time", output_map)
-            self.assertEqual(output_map["time"]["source"], "HARDCODED")
-            self.assertEqual(output_map["time"]["value"], False)
-            self.assertEqual(output_map["time"]["history"], [])
-            for _, val in output_map.items():
-                self.assertIn("history", val)
-        except ValueError:
-            self.fail("Invalid JSON output")
-
-    def test_options_option(self) -> None:
-        pants_run = self.run_pants(
-            ["options", "--no-colors", "--name=colors", "--no-skip-inherited"]
-        )
-        pants_run.assert_success()
-        self.assertIn("options.colors = ", pants_run.stdout)
-        self.assertIn("pytest.colors = ", pants_run.stdout)
-        self.assertNotIn("options.scope = ", pants_run.stdout)
-
-    def test_options_only_overridden(self) -> None:
-        pants_run = self.run_pants(["options", "--no-colors", "--only-overridden"])
-        pants_run.assert_success()
-        self.assertIn("options.only_overridden = True", pants_run.stdout)
-        self.assertNotIn("options.scope =", pants_run.stdout)
-        self.assertNotIn("from HARDCODED", pants_run.stdout)
-        self.assertNotIn("from NONE", pants_run.stdout)
-
-    def test_options_rank(self) -> None:
-        pants_run = self.run_pants(["options", "--no-colors", "--rank=FLAG"])
-        pants_run.assert_success()
-        self.assertIn("options.rank = ", pants_run.stdout)
-        self.assertIn("(from FLAG)", pants_run.stdout)
-        self.assertNotIn("(from CONFIG", pants_run.stdout)
-        self.assertNotIn("(from HARDCODED", pants_run.stdout)
-        self.assertNotIn("(from NONE", pants_run.stdout)
-
-    def test_options_show_history(self) -> None:
-        pants_run = self.run_pants(
-            ["options", "--no-colors", "--only-overridden", "--show-history"]
-        )
-        pants_run.assert_success()
-        self.assertIn("options.only_overridden = True", pants_run.stdout)
-        self.assertIn("overrode False (from HARDCODED", pants_run.stdout)
-
-    def test_from_config(self) -> None:
```

Comment:
This seemed redundant to test with all of our unit tests. Why test loading from a config file, but not an env var, for example?